packages/yoga/src/DropdownInput/web/DropdownInput.jsx

```diff
@@ -0,0 +1,317 @@
+import React, { useRef, useState } from 'react';
+import PropTypes, { bool } from 'prop-types';
+import { theme, Box, Icon, Text } from '@gympass/yoga';
+import styled, { css } from 'styled-components';
+import { Check, ChevronUp, ChevronDown, Close } from '@gympass/yoga-icons';
+import { display, flexes } from '@gympass/yoga-system';
+import { useOnClickOutside } from '../../hooks';
+
+const Wrapper = styled.div`
+  ${({
+    full,
+    theme: {
+      yoga: {
+        components: { dropdowninput },
+      },
+    },
+  }) => `
+    position: relative;
+    width: ${full ? '100%' : `${dropdowninput.width}px`};
+  `}
+`;
+
+const Container = styled.div`
+  ${({
+    theme: {
+      yoga: {
+        components: { dropdowninput },
+      },
+    },
+  }) => `
+    ${display}
+    ${flexes}
+    height: 52px;
+    box-sizing: border-box;
+    transition: all 0.1s ease-in-out;
+    border-style: solid;
+    border-radius: ${dropdowninput.container.border.radius}px;
+    border-width: ${dropdowninput.container.border.width}px;
+    border-color: ${dropdowninput.container.border.color};
+    background: ${dropdowninput.container.background};
+    margin-top: ${dropdowninput.container.margin.top}px;
+    ${({ showDropDown }) =>
+      showDropDown &&
+      css`
+        border-width: ${dropdowninput.container.border.width}px
+          ${dropdowninput.container.border.width}px 0
+          ${dropdowninput.container.border.width}px;
+        border-radius: ${dropdowninput.container.border.radius}px
+          ${dropdowninput.container.border.radius}px 0 0;
+      `}
+  `}
+`;
+
+const Clear = styled.button`
+  ${({
+    theme: {
+      yoga: {
+        components: { dropdowninput },
+      },
+    },
+  }) => `
+    background: ${dropdowninput.clear.background};
+    border: ${dropdowninput.clear.border};
+    cursor: ${dropdowninput.clear.cursor};
+    display: ${dropdowninput.clear.display};
+    outline:  ${dropdowninput.clear.outline};
+    padding: ${dropdowninput.clear.padding}
+  `}
+`;
+
+const Input = styled.input`
+  background: none;
+  border: none;
+  color: ${theme.colors.text.primary};
+  flex: 1;
+  font-family: ${({ theme: { yoga } }) => yoga.baseFont.family};
+  font-size: ${theme.fontSizes.small}px;
+  font-style: normal;
+  font-weight: ${theme.fontWeights.regular};
+  height: 100%;
+  letter-spacing: 0px;
```

Comment:
I noticed that you used `0` in the other properties and here you used `0px,` I should you keep `0`

Reply:
✅